sw-emulator/lib/periph/src/hmac_sha384.rs

```diff
@@ -484,8 +482,8 @@ impl HmacSha384 {
             ),
         };
 
-        if data != None {
-            self.format_block(&data.unwrap());
+        if let Some(ref data) = data {
+            self.format_block(data);
```

Comment:
nit: instead of `ref`, we could borrow at the argument
`self.format_block(&data)`

Reply:
Done